File: src/Generated20260113.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260113 {
public:
    // Generated class 20260113_205012_580737
    int value = 903;
    string created = "20260113_205012_580737";
    
    int process() {
        int result = value;
        for (int i = 0; i < 25; i++) {
            result += i * 2;
        }
        return result;
    }
};

int main() {
    Generated20260113 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260113.go
package main

import "fmt"

type Generated20260113 struct {
    // Generated struct 20260113_204919_395819
    Value int
    Created string
}

func (c *Generated20260113) Process() int {
    result := c.Value
    for i := 0; i < 8; i++ {
        result += i * 9
    }
    return result
}

func main() {
    obj := Generated20260113{Value: 891, Created: "20260113_204919_395819"}
    fmt.Println(obj.Process())
}